webdriver/tests/support/image.py

```diff
@@ -6,6 +6,7 @@
 
 PPI = 96
```

Comment:
Maybe you can rename that one as well while we are modifying this code? It's only used at two places. It should be `inch_in_pixel`.